security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -27,6 +27,11 @@ import (
 // The size of a private key for a leaf certificate.
 const keySize = 2048
 
+// SecretStore defines interface that secret storages need to implement.
```

Comment:
Could you rephrase it into what SecretStores provides more specifically? The interface should states the user expected functionality, and disregard the implementor.

Reply:
renamed and updated.